content/react/en/addons.md

```diff
@@ -0,0 +1,116 @@
+---
+title: "Addons"
+tocTitle: "Addons"
+description: "Learn how to integrate and use a very popular addon"
+---
+
+# Addons
+
+Storybook boasts a robust system of [addons](https://storybook.js.org/addons/introduction/) with which you can enhance the developer experience for
+everybody in your team. If you've been following along with this tutorial linearly, we have referenced multiple addons so far, and you will have already implemented one in the [Testing chapter](/test).
+
+<div class="aside">
+<strong>Looking for a list of potential addons?</strong>
+<br/>
+😍 You can seen the list of officially-supported and strongly-supported community addons <a href="https://storybook.js.org/addons/addon-gallery/">here</a>.
+</div>
+
+We could write forever about configuring and using addons for all of your particular use-cases. For now, let's work towards integrating one of the most popular addons within Storybook's ecosystem: [knobs](https://github.com/storybooks/storybook/tree/v3.3.15/addons/knobs).
+
+<div class="aside">
+<strong>⚠ Version 3.3.15 of Knobs ⚠</strong>
```

Comment:
Was there a reason for this particular version?

Reply:
I felt weird about this one. I just wanted to be crystal clear about matching up with the dependencies outlined in the example code base. The installation and usage look to be the same on master, so I'll revert the version-specifying content.